test/integration/resultsPagePlace.js

```diff
@@ -80,7 +80,7 @@ describe('The place results page', () => {
         const $ = cheerio.load(res.text);
 
         expect($('.results__header').text())
-          .to.include('There are 3 places matching multiresult');
+          .to.include('We found 3 places that match \'multiresult\'');
```

Comment:
Is it worth extracting `multiresult` to a variable? It is used 3 times within this test.